stable/newrelic-infrastructure/README.md

```diff
@@ -8,16 +8,18 @@ This chart will deploy the New Relic Infrastructure agent as a Daemonset.
 
 | Parameter                 | Description                                                  | Default                    |
 | ------------------------- | ------------------------------------------------------------ | -------------------------- |
-| `cluster`                 | The cluster name for the Kubernetes cluster.                 |                          |
+| `cluster`                 | The cluster name for the Kubernetes cluster.                 |                            |
 | `licenseKey`              | The [license key](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key)  for your New Relic Account. | |
-| `config`                  | A `newrelic.yml` file if you wish to provide.                |                         |
+| `customSecret.name`       | Same as `licenseKey` but specified as custom secret          |                            |
+| `customSecret.key`        | Same as `licenseKey` but specified as custom secret          |                            |
```

Comment:
I think we can improve the description of these 2 secrets. These are my suggestions, but feel free to improve them:

`customSecret.name`: name of the Secret object where the license key is stored.
`customSecret.key`: key in the Secret object where the license key is stored.

Reply:
will do